patches/element-ui@2.15.12.patch

```diff
@@ -0,0 +1,263 @@
+diff --git a/lib/element-ui.common.js b/lib/element-ui.common.js
```

Comment:
Does it makes sense to send this patch as an PR to upstream,so that we don't have to maintain this?

Reply:
Is this patch from [this PR](https://github.com/ElemeFE/element/pull/20671)?

Reply:
Yes

Reply:
[Looking at the releases, they are actively merging into 2.x](https://github.com/ElemeFE/element/releases).

Reply:
Maybe one day :D

Reply:
I haven't actually tested this yet. but, since @cstuncsik approved it, please feel free to merge.